id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -241,8 +241,7 @@ public void validateRequest(Object request, Errors errors, String method) {
 						String schemaVersion;
 						if (requestMap.get(ROOT_PATH) != null) {
 							schemaVersion = String
-									.valueOf(((Map<String, Object>) requestMap.get(ROOT_PATH))
```

Comment:
I think this is not related to resident change. Please keep it if it is there in release.

Reply:
Done